tests/integration/test_dbfs_store/test-dbfs-store.yml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+env:
+  # databricks authentication token
+  DATABRICKS_TOKEN:
+  # databricks endpoint - for example abc-d1e2345f-a6b2.cloud.databricks.com (this example is from docs.databricks.com
+  # - it is not for use)
+  DATABRICKS_WORKSPACE:
```

Comment:
How did `DATABRICKS_HOST` become `DATABRICKS_WORKSPACE`? If you use the correct environment variable, you won't have to pass it around.

Reply:
I can pass it around or save it as an environment variable. I think both ways are good.

Reply:
And renamed to DATABRICKS_HOST.

Reply:
This yml defines the env, so passing `DATABRICKS_HOST` around in the tests is completely unnecessary now.